src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -67,6 +67,17 @@ public FeedOptions FeedOptions
 
             var temporaryProjectFileFullPath = CreateTemporaryProject(FeedOptions.TargetPackagesFolder, packageId, packageVersion);
 
+            var packageFullPath = Path.Combine(
```

Comment:
probably we should change the help text too? currently it says `Overwrites conflicting commands`